samples/matter/light_bulb/boards/nrf21540dk_nrf52840.overlay

```diff
@@ -27,7 +27,7 @@
 
 	pwmleds {
 		pwm_led1: pwm_led_1 {
-			pwms = < &pwm0 0xe >;
+			pwms = <&pwm0 1 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
```

Comment:
I verified Matter samples empirically, and I found that the pwm_led logic had been changed. When I'm sending the command to switch the LED on, the PWM is set to 0; after the command to switch the LED off, the PWM is set to the maximum value.

Reply:
Fixed.